.eslintrc.yaml

```diff
@@ -83,7 +83,7 @@ rules:
     - warn
     - code: 150
   # no-array-constructor: off
-  no-console: error
```

Comment:
I don't believe we want to allow console statements in the code. Perhaps this was disabled during testing and needs to be re-enabled?